<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,7,2,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="m_SelectSavePathBtn">
        <property name="text">
         <string>选择保存路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_DisplaySavePathLineEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="m_DisplayModeComboBox">
        <item>
         <property name="text">
          <string>单栏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>双栏</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_SizeComboBox">
        <item>
         <property name="text">
          <string>最佳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>全屏</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="m_SleepSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="m_DisplayCaptureLabel">
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="m_DisplayCaptureFramesLabel">
        <property name="text">
         <string>采集xx帧</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_DisplayCaptureTimeLabel">
        <property name="text">
         <string>用时xx秒</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_LinkBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_StartBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_StopBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
